emails/payin_directdebit_failed.spt

```diff
@@ -1,13 +1,13 @@
-{{ _("The direct debit payment to your Liberapay wallet has failed") }}
+{{ _("The direct debit payment to your Liberapay wallet failed") }}
```

Comment:
has is redundant as failed is in the past already

Reply:
The present perfect simple expresses an action that is still going on or that **stopped recently**.